core/src/main/kotlin/in/specmatic/core/pattern/models/NegativePatternConfiguration.kt

```diff
@@ -0,0 +1,5 @@
+package `in`.specmatic.core.pattern.models
```

Comment:
instead of `pattern.models`, can you call it `pattern.config`.

Reply:
done